fitbenchmarking/controllers/tests/test_basic_controllers.py

```diff
@@ -1,3 +1,4 @@
+# Tests for controllers available from basic fitbenchmarking install
```

Comment:
We should probably use `default` instead of `basic` here and for the file name too